### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewDebug_ColorContrastAnalyzer: Color Contrast ratio for names of the toolbars is less than required in Light VS theme.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52309]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "View Debug "(Cntrl+Shift+D) button.
3. Navigate to toolbars section under heading "Debug".
3. Apply Light Visual Studio Theme.
4. Start Color Contrast Analyzer to determine the color contrast ratio of names of the toolbars.

**Actual:**
Color Contrast ratio of the names of the toolbars is less than 4.5:1 in Light VS theme.

**Expected:**
Color Contrast ratio of names of the toolbars should be >=4.5:1 in Light VS theme.

**Recommendations:**
Adjust color of that element to meet the minimum color contrast requirement. 
Or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Low visioned users will not clearly understand the toolbar text written on the screen.

**MAS Reference:**
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment For Reference:**

![a11y_vscode_viewdebug_colorcontrastanalyzer](https://user-images.githubusercontent.com/39204909/41591086-66bc3fb4-73d6-11e8-9b40-87cdc610023b.png)

Does this issue occur when all extensions are disabled?: Yes
